packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -1422,7 +1423,7 @@ Future<T?> showCupertinoDialog<T>({
       context: context,
       barrierDismissible: barrierDismissible,
       barrierLabel: barrierLabel,
-      barrierColor: CupertinoDynamicColor.resolve(kCupertinoModalBarrierColor, context),
```

Comment:
maybe this should be `barrierColor ?? CupertinoDynamicColor.resolve(kCupertinoModalBarrierColor, context)`?

Reply:
It should be because `CupertinoDialogRoute`'s `barrierColor` defaults to the same value.

Reply:
> It should be because CupertinoDialogRoute's barrierColor defaults to the same value.

Yes, that’s the reason it was removed